src/makeRobot.js

```diff
@@ -38,7 +38,56 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${this.name}, chip version:
+ ${this.version}, wheels: ${this.wheels}`.replace(/(\r\n|\n|\r)/gm, '');
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward: function(i = 1) {
+      if (i >= 0) {
+        this.coords.y += i;
```

Comment:
```suggestion
    goForward: function(step = 1) {
      if (step >= 0) {
        this.coords.y += step;
```

И аналогично в других методах